kinto/views/__init__.py

```diff
@@ -0,0 +1,26 @@
+import random
+import string
+
+from cliquet.storage import generators, exceptions
+from pyramid import httpexceptions
+
+
+class NameGenerator(generators.Generator):
+    def __call__(self):
+        try:
+            ascii_letters = string.ascii_letters
+        except ImportError:
+            ascii_letters = string.letters  # NOQA
+        alphabet = ascii_letters + string.digits + '-'
+        letters = [random.choice(alphabet) for x in range(8)]
+        return ''.join(letters)
```

Comment:
This doesn't prevent merging of this, but I think the name generation should be pluggable. Having an id generator is something we had in daybed and I think we should continue like this: it helps users to generate whatever they want as a new name. for instance the koremutake generator for daybed @natim pointed.